docs/workflows/oci-artifacts.rst

```diff
@@ -0,0 +1,54 @@
+Managing additional OCI media types
+===================================
+
+.. _default-oci-types:
+
+By default the following list of media types is enabled in the Container Registry::
+
+ * OCI images
+ * Helm
+ * Cosign, SBOMs, attestations
+ * Source containers
+ * Singularity
+ * Conftest policies
+ * WASM
+
+For any other OCI media type that is not supported by default, you can add them to the
+ADDITIONAL_OCI_ARTIFACT_TYPES settings using the following format::
+
+ ADDITIONAL_OCI_ARTIFACT_TYPES = {
+    "<oci config type 1>": [
+        "<oci layer type A>",
+        "<oci layer type B>",
+    ],
+    "<oci config type 2>": [
+        "<oci layer type C>",
+        "<oci layer type D>",
+    ],
+ }
+
+
+For example, you can add support for custom defined mediatype by adding the following to your
+ADDITIONAL_OCI_ARTIFACT_TYPES setting::
+
+ ADDITIONAL_OCI_ARTIFACT_TYPES = {
+    "<oci config type 1>": [
+        "<oci layer type A>",
+        "<oci layer type B>",
+    ],
+    "<oci config type 2>": [
+        "<oci layer type C>",
+        "<oci layer type D>",
+    ],
+    "application/vnd.guardians.groot.config.v1+json": [
+        "text/plain",
+        "application/vnd.guardians.groot.docs.layer.v1+tar",
+    ],
+ }
+
+ .. note::
+
+When adding OCI media types that are not configured by default, users will also need to manually add
+support for the :ref:`Default oci types<default-oci-types>`.
```

Comment:
Can we remove this sentence? The whole chapter is about adding new media types.

Reply:
I re-phrased